app/Http/Middleware/CacheControl.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of Cachet.
+ *
+ * (c) Alt Three Services Limited
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace CachetHQ\Cachet\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+
+class CacheControl
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @param \Closure                 $next
+     *
+     * @return mixed
+     */
+    public function handle(Request $request, Closure $next)
+    {
+        $response = $next($request);
+
+        $maxAge = time() + 30;
+
+        $response->header('Cache-Control', 'public,max-age='.$maxAge);
```

Comment:
I wonder if it's also worth sending a `Last-Modified` date, and handling requests with an `If-Modified-Since` header (return `304 Not Modified` if the content hasn't changed). Perhaps that's overkill though, and it'd require a bit more work, as ideally you'd want to bail out of most of the work if you can quickly determine that there have been no changes since the specified date.

Reply:
`Last-Modified` could be worked out by the latest component or incident that affected the status, yes.

Reply:
@Daniel15 actually, what I said above is a bit more complicated as we don't store the global state anywhere, so we'd have to re-fetch the components in the middleware. As you say, this may be overkill anyway, so I'm going to merge this as-is.